### ISSUE TITLE
List code style violations

### ISSUE THREAD

**Issue by MrAnyx** [PHP-CS-Fixer/PHP-CS-Fixer#6743]
## Support question

I used to use [squizlabs/PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer)  but in the end I find PHP-CS-Fixer easier to configure and the integration with VSCode is really great.

The only thing I miss is a way to list all code style violations in all included files (like the phpcs cli). I found the `list-files` command but it only lists all the included files, not the violations.

Is there any way to replicate the behavior of phpcs with a command like `php php-cs-fixer.phar check`? So that I can use this command in a Github action for example and check the code style of a pull request.

**Comment by kubawerlos** [comment]
if only there was a place where the [usage](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.13.2/doc/usage.rst) of the fixer is described.

Or if it was possible it could be added to [CI of the tool](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.13.2/.github/workflows/ci.yml#L150) itself.

I always wonder when I see an issue like this - what did you do to find it before you raised it?

**Comment by MrAnyx** [comment]
> if only there was a place where the [usage](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.13.2/doc/usage.rst) of the fixer is described.
> 
> Or if it was possible it could be added to [CI of the tool](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.13.2/.github/workflows/ci.yml#L150) itself.
> 
> I always wonder when I see an issue like this - what did you do to find it before you raised it?

**Comment by keradus** [comment]
TL;DR - `fix --dry-run` command

**Comment by kubawerlos** [comment]
> TL;DR - `fix --dry-run` command

@MrAnyx if you add `--diff -vvv` to it, there is a diff of what would be fixed and the list of fixers applied, per file.

If you only click any of the 2 links I've posted in 1st reply you would know.

**Comment by MrAnyx** [comment]
Indeed, I missed the doc about `--dry-run`. It seems to do the job. Thank you @keradus
